packages/utils/src/lib/reports/formatting.unit.test.ts

```diff
@@ -112,3 +118,108 @@ describe('metaDescription', () => {
     );
   });
 });
+
+describe('formatSourceLine', () => {
+  it.each([
+    [{ startLine: 2 }, '2'],
+    [{ startLine: 2, endLine: undefined }, '2'],
+    [{ startLine: 2, endLine: 2 }, '2'],
+    [{ startLine: 2, endLine: 3 }, '2-3'],
+  ])('should format position %o as "%s"', (position, expected) => {
+    expect(
+      formatSourceLine({ file: '/packages/utils/src/index.ts', position }),
+    ).toBe(expected);
+  });
+
+  it('should return an empty string when position is missing', () => {
+    expect(formatSourceLine({ file: '/packages/utils/src/index.ts' })).toBe('');
+  });
+});
```

Comment:
Wouldn't it make more sense for the `formatSourceLine` to receive only the position? The source object includes irrelevant properties like `file`.